doc/tutorials/introduction/linux_install/linux_install.markdown

```diff
@@ -21,13 +21,39 @@ Required Packages
 -   [optional] libjpeg-dev, libpng-dev, libtiff-dev, libjasper-dev, libdc1394-22-dev
 -   [optional] CUDA Toolkit 6.5 or higher
 
-The packages can be installed using a terminal and the following commands or by using Synaptic
-Manager:
+The packages can be installed using a terminal and the following commands or by using Synaptic Manager in Ubuntu 10.04, 16.04 LST:
 @code{.bash}
 [compiler] sudo apt-get install build-essential
 [required] sudo apt-get install cmake git libgtk2.0-dev pkg-config libavcodec-dev libavformat-dev libswscale-dev
 [optional] sudo apt-get install python-dev python-numpy libtbb2 libtbb-dev libjpeg-dev libpng-dev libtiff-dev libjasper-dev libdc1394-22-dev
 @endcode
+
+The packages can be installed using a terminal and the following commands or by using Synaptic Manager in Ubuntu 18.04:
+@code{.bash}
+[compiler] sudo apt-get install build-essential
+[required] sudo apt-get install cmake git libgtk-3-dev pkg-config libavcodec-dev libavformat-dev libswscale-dev
```

Comment:
`libav*` and `libgtk*` libraries are not `required`, `git` is also optional.

Same comment applies to other platforms below.

Reply:
> `libav*` and `libgtk*` libraries are not `required`, `git` is also optional.
No @mshabunin, libav* command will install other unnecessary libraries.For example (libavfs0, libaviplay, libavahi and other.
> libraries are not required, git is also optional.
I followed the original documentation, but I agree with you. But without git, the ***git clone*** command will not work.